packages/shorebird_cli/lib/src/commands/subscription/subscription_command.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/commands/subscription/subscription.dart';
+
+/// {@template subscription_command}
+///
+/// `shorebird subscription`
+/// Manage your Shorebird subscription.
+/// {@endtemplate}
+class SubscriptionCommand extends ShorebirdCommand {
```

Comment:
How does this relate to "account"?

Reply:
I was thinking about this as well

Reply:
I support your bias toward action (just landing a thing). :)